src/lms-ds-loader/edfi_lms_ds_loader/scripts/postgresql/0002_create_user_tables.sql

```diff
@@ -4,39 +4,57 @@
 -- See the LICENSE and NOTICES files in the project root for more information.
 
 CREATE TABLE lms.LMSUser (
-    LMSUserIdentifier SERIAL,
+    LMSUserIdentifier INT GENERATED ALWAYS AS IDENTITY,
```

Comment:
This is a newer, preferred style. Luis started using it in other files, and I've backported it to this one.